### DISCUSSION TITLE
World of Light Freezes

### DISCUSSION THREAD

**Discussion by ghost** [Raytwo/ARCropolis#598] [Issues]
As of updating Smash Ultimate to 13.0.3 and Updating Arcropolis. World of Light doesn’t work properly.
In the main world if i try to go to another area it will freeze, if i try to open the menu it will freeze, if i start a battle it will freeze. This didn’t occur on the previous Arcropolis version and SSBU 13.0.2

Occuring on real Switch. My modded Switch is running on 18.1.0

**Comment by Raytwo** [comment]
Ok sure but you're not telling us much here, what ARCropolis version is this. There have been two since 13.0.3

**Reply by ghost** [reply]
Arcropolis version 4.0.4 is used.
Super Smash Bros Ultimate is running on 13.0.3
